ctapipe/image/muon/processor.py

```diff
@@ -36,10 +36,16 @@ class ImageParameterQuery(QualityQuery):
     quality_criteria = List(
         Tuple(Unicode(), Unicode()),
         default_value=[
-            ("min_pixels", "p.morphology.n_pixels > 100"),
-            ("min_intensity", "p.hillas.intensity > 500"),
+            ("min_pixels", "dl1_params.morphology.n_pixels > 100"),
+            ("min_intensity", "dl1_params.hillas.intensity > 500"),
         ],
-        help=QualityQuery.quality_criteria.help,
+        help=(
+            "list of tuples of ('<description', 'expression string') to accept "
```

Comment:
I would explicitly write something like "Quality cuts as ... to select muon images for analysis...".

Reply:
As in:

```
Quality cuts given as list of tuples of ('description', 'expression string'), e.g. ``[('mycut', 'x > 3'),]``, 
to select muon images for further analysis. 
You may use ``numpy`` as ``np`` and ``astropy.units`` as ``u``, but no other modules. 
DL1 image parameters can be accessed by prefixing the wanted parameter with ``dl1_params``.
```

?